### PR TITLE
Added an important note.

### PR DESCRIPTION
**Pull request by j0rt3g4**
issue #2922 Added a note about supportability.

I think is good either way...
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

### CODE REVIEW

File: Skype/SfBServer2019/plan/system-requirements.md [hunk 1]
```diff
@@ -101,6 +101,9 @@ Anything other than the operating systems listed here won't work properly; pleas
 <a name="Software"> </a>
 
 There are some things you're going to need to install or configure for any server running Skype for Business Server 2019. These are listed below, followed by additional requirements for specific server roles.
+
+> [!IMPORTANT]
+> .Net Framework 4.8 is still not supported with Skype For Business 2019.
```
Comment (JohanFreelancer9) [inline Skype/SfBServer2019/plan/system-requirements.md]:
```suggestion
> .Net Framework 4.8 is not yet supported by Skype for Business Server 2019.
```

File: Skype/SfBServer2019/plan/system-requirements.md [hunk 2]
```diff
@@ -101,6 +101,9 @@ Anything other than the operating systems listed here won't work properly; pleas
 <a name="Software"> </a>
 
 There are some things you're going to need to install or configure for any server running Skype for Business Server 2019. These are listed below, followed by additional requirements for specific server roles.
+
+> [!IMPORTANT]
+> .Net Framework 4.8 is not supported by Skype for Business Server 2019.
```
Comment (nenonix) [inline Skype/SfBServer2019/plan/system-requirements.md]:
```suggestion
> .NET Framework 4.8 is not supported by Skype for Business Server 2019.
```

**Comment by jambirk** [comment]
This is not factually correct. Closing this PR,

**Comment by j0rt3g4** [comment]
@jambirk  if it's not correct, then do comment about what is correct, don't just close it that that's it.
So I can fix the PR and the Issue.
@scanum

**Pull request closed without merging**